### PR TITLE
Future proof static hours parsing

### PR DESCRIPTION
**Pull request by ilikesymmetry**

### CODE REVIEW

File: src/data.py [hunk 1]
```diff
@@ -227,7 +227,14 @@ def parse_eatery_type(eatery):
 def parse_static_op_hours(hours_list, eatery_id, dining_items):
   weekdays = {}
   for hours in hours_list:
-    for weekday in WEEKDAYS[hours['weekday']]:
+    if '-' in hours['weekday']:
```
Comment (alanna-zhou) [inline src/data.py:230]: why would there be hyphens in the hours?

File: src/constants.py [hunk 2]
```diff
@@ -26,13 +26,13 @@
...
 # [0, 1, 2, 3, 4, 5, 6]
 WEEKDAYS = {
-    'monday-wednesday': [0, 1, 2],
-    'monday-thursday': [0, 1, 2, 3],
-    'monday-friday': [0, 1, 2, 3, 4],
-    'thursday': [3],
-    'friday': [4],
-    'saturday': [5],
-    'sunday': [6]
+    'monday': 0,
```
Comment (alanna-zhou) [inline src/constants.py:29]: how come this got changed from monday-wednesday to monday?

**Review by alanna-zhou — APPROVED** [review]
looks okay to me just curious about some things

File: src/data.py (hunk 1 above)
Comment (ilikesymmetry) [reply src/data.py:230]: in the externalEateries file, we represent days as 'monday-wednesday' or just 'wednesday' for example.

File: src/constants.py (hunk 2 above)
Comment (ilikesymmetry) [reply src/constants.py:29]: the previous method of having 'monday-...' correspond to a list of indexes would require an exhaustive list of day combinations to include all edge cases, having the days correspond to an index and then make the list in a method is more streamlined approach.

**Pull request merged**